src/components/dictionaryConcepts/components/CreateMapping.jsx

```diff
@@ -16,6 +16,8 @@ class CreateMapping extends Component {
     this.setState({ inputValue: value });
   }
 
+  sourceCaseSensitivity = source => source.trim().toUpperCase();
```

Comment:
I think `sourceCaseSensitivity` can be given a name that better tells what it does.

Reply:
I have changed it to a better name `sourceNameToUpperCase`